internal/pkg/template/diff/diff_test.go

```diff
@@ -353,8 +337,65 @@ Dog: (pleased) "ikr"`, t),
 			got, err := From(tc.old).Parse([]byte(tc.curr))
 			if tc.wantedError != nil {
 				require.EqualError(t, err, tc.wantedError.Error())
+			} else {
+				require.NoError(t, err)
+				require.True(t, equalTree(got, Tree{tc.wanted()}, t), "should get the expected tree")
 			}
-			if tc.wanted != nil {
+		})
+	}
+}
+
+func TestFrom_Parse_CFN(t *testing.T) {
```

Comment:
Just for consistency does it have to be?
```suggestion
func TestFrom_ParseWithCFNIgnorer(t *testing.T) {
```